src/components/tl/mlx5/tl_mlx5_coll.h

```diff
@@ -12,6 +12,7 @@
 
 typedef struct ucc_tl_mlx5_task {
     ucc_coll_task_t super;
+    void                       *req_handle;
```

Comment:
since it's specific to mcast we usually add this as union e.g.
```
typedef struct ucc_tl_mlx5_task {
    ucc_coll_task_t super;
    union {
        struct bcast_mcast{
            void *req_handle
        }
   }
}
```